api/server/groups.go

```diff
@@ -14,22 +14,17 @@ import (
 	"net/http"
 
 	log "github.com/Sirupsen/logrus"
-
-	"github.com/Mierdin/todd/db"
 )
 
 func (tapi ToDDApi) Groups(w http.ResponseWriter, r *http.Request) {
 
 	log.Info("Received request for group map")
 
-	var tdb = db.NewToddDB(tapi.cfg)
-
-	groupmap := tdb.DatabasePackage.GetGroupMap()
-
-	// If there are no objects, return an empty slice, not a nil slice - this
-	// prevents this API from returning "null"
-	if groupmap == nil {
```

Comment:
NOTE - I am just wanting to make sure that this change doesn't undo what the comment describes. I need to test ToDD to ensure that the API still returns a ```{}``` instead of ```null``` when there is nothing in the group map